src/server/utilities.ts

```diff
@@ -103,24 +103,6 @@ namespace ts.server {
         }
     }
 
-    export function removeItemFromSet<T>(items: T[], itemToRemove: T) {
-        if (items.length === 0) {
-            return;
-        }
-        const index = items.indexOf(itemToRemove);
-        if (index < 0) {
-            return;
-        }
-        if (index ===  items.length - 1) {
-            // last item - pop it
-            items.pop();
-        }
-        else {
-            // non-last item - replace it with the last one
-            items[index] = items.pop();
-        }
-    }
-

```

Comment:
Looks like there was never any test for `removeItemFromSet`?
Are there tests for `unorderedRemoveItem`?